.dockerignore

```diff
@@ -4,4 +4,7 @@ vendor/pkg
 .go-pkg-cache
 .git
 hack/integration-cli-on-swarm/integration-cli-on-swarm
+bundles
+Makefile
```

Comment:
?

Reply:
`bundles` should not be considered in the build cache (otherwise you need to rebuild with each invocation), and `Makefile` is not used except on the client side since it is all assuming you want to start up a container.